### DISCUSSION TITLE
Cursor tracking skipping

### DISCUSSION THREAD

**Discussion by IAmY00** [OpenTabletDriver/OpenTabletDriver#2566] [Q&A]
## Description

My XP-Pen Star G640 connects to the software, and as far as I can tell, there are no other drivers installed apart from OTD, but my cursor skips about 2-3 inches in certain areas depending on how large my area is, and where it is located within the tablet's dimensions. If the area is full size, there are no problems what so ever. I tried searching for XP-Pen's PenTablet driver in processes, services, and details without success. I honestly think it's duplicate detection from multiple drivers, or a scaling issue. I have a work around, but it prohibits me from using my original area size and or location in relation to the tablet's physical dimensions. Debug and diagnostic reports seemed fine according to mods and helpers at the time.
## System Information: Windows 10 Version 21H2

| Name             | Value |
| ---------------- | ----- |
| Operating System| Windows 10 Version 21H2
| OpenTabletDriver Version| v0.6.0.2 AND v0.5.3.3
| Tablet| XP-Pen Star G640

**Comment by jamesbt365** [comment]
if this happens in the tablet > tablet debugger this is a hardware issue.
